scala/scala.bzl

```diff
@@ -147,7 +137,7 @@ rm -rf {out}_tmp
     outs.extend([ctx.outputs.ijar])
   ctx.action(
       inputs=list(jars) +
-          list(dep_srcjars) +
+          list(all_srcjars) +
           ctx.files.srcs +
```

Comment:
shouldn't this be `sources` now? You partitioned that into `sources` and `srcjars`.